wpa_supplicant/wpa_cli_cmds.c

```diff
@@ -3014,6 +3012,31 @@ static const struct wpa_cli_cmd wpa_cli_commands[] = {
 	{ "interface_list", wpa_cli_cmd_interface_list, NULL,
 	  cli_cmd_flag_none,
 	  "= list available interfaces" },
+#ifdef CONFIG_AP
```

Comment:
Why do we move this block around, this just makes it harder when working with upstream?

Reply:
This was under the `#ifndef !CONFIG_ZEPHYR`, so, I had to move it out of that block.

Reply:
Ok, understood. But if this is to be proposed to upstream and/or when getting a new upstream version, it would be better if we make smaller changes as it helps with conflict resolving. Could we adjust the ifdef statements so that we do not need to move the code?

Reply:
Upstream doesn't even have this file, I have carve it out of `wpa_cli.c` to avoid duplication.

> Could we adjust the ifdef statements so that we do not need to move the code?

It's tricky, as its in the middle as below.
```
#ifndef !CONFIG_ZEPHYR
..
..
...
#ifdef CONFIG_AP
...
...
#endif
....
...
#endif
```

Reply:
> Upstream doesn't even have this file, I have carve it out of `wpa_cli.c` to avoid duplication.

Ok, I thought it had because I did not see "zephyr" mentioned in the file name. No worries then, I was mostly concerned about the future merges.